src/taco-cli/cli/utils/cordovaWrapper.ts

```diff
@@ -188,6 +190,24 @@ class CordovaWrapper {
         options: { logLevel?: tacoUtility.InstallLogLevel, isSilent?: boolean, captureOutput?: boolean } = {}): Q.Promise<T | string> {
         return CordovaHelper.tryInvokeCordova<T | string>(apiFunction, () => CordovaWrapper.cli(processArgs(), options.captureOutput), options);
     }
+
+    /**
+     * Returns a path to the local Cordova command if ran from a taco project, otherwise it will return the
+     * global Cordova command.
+     */
+    private static getCordovaCommand(): Q.Promise < string > {
+        return CordovaHelper.tryInvokeCordova<string>((cordova: typeof Cordova): string => {
+        return (<{ packageTargetPath?: string }> cordova).packageTargetPath;
```

Comment:
Not every package has an executable though, while every package does have an install location. Do we want to tell TacoPackageLoader whether a package is executable (and what the name is), or should we leave that to the caller? I suppose we could read the package.json of the installed package and check for any "bin" entries.